core-bundle/src/Twig/Loader/Theme.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Loader;
+
+use Contao\CoreBundle\Exception\InvalidThemePathException;
+use Webmozart\PathUtil\Path;
+
+class Theme
```

Comment:
Is there maybe a better name that is not so generic? Maybe `ThemeNamespace` or `ThemeNamespaces`?

Reply:
renamed in 79912131d608bc47ae26a1bb055b9ac6d6e1f105